RecoParticleFlow/PFClusterProducer/python/particleFlowClusterECAL_cfi.py

```diff
@@ -1,11 +1,178 @@
 import FWCore.ParameterSet.Config as cms
 
+#temporarily load regression weights with esprefer
+from CondCore.DBCommon.CondDBCommon_cfi import *
+
+pfcluscorsource = cms.ESSource("PoolDBESSource",
+    CondDBCommon,
+    DumpStat=cms.untracked.bool(False),
+    toGet = cms.VPSet(
+      cms.PSet(
+        record = cms.string('GBRDWrapperRcd'),
+        label = cms.untracked.string('GBRForestD_ecalPFClusterCor_EB_pfSize1_mean_50ns'),
+        tag = cms.string('GBRForestD_ecalPFClusterCor_EB_pfSize1_mean_50ns')
```

Comment:
instead of adding a new PoolDBESSource module, you could add them directly to the global tag module (at least, years ago I was told that having two PoolDBESSource modules in the same configuration is Bad).

If you prefer to keep these in their own PoolDBESSource module, make sure that cmsDriver loads it together with the other condtions.